src/atoms/StudentImage/StudentImage.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+import { Colors, Shadow } from 'context';
+
+import { getInitials } from './utils/getInitials';
+
+const { mainColor, secondaryColor } = Colors;
+const { shadow } = Shadow;
+
+export interface IStyledStudentImageProps {
+  userPicture?: string,
+  firstName: string,
+  lastName: string,
+}
+
+export interface IStudentImageProps extends IStyledStudentImageProps {
+  isInCircle: boolean,
+}
+
+const StyledStudentImage = styled.div<IStudentImageProps>`
```

Comment:
Just create two separate components (Circle and not circle) and choose based on isInCirle one of them